akka-actor-tests/src/test/scala/akka/event/EventStreamSpec.scala

```diff
@@ -394,6 +438,7 @@ class EventStreamSpec extends AkkaSpec(EventStreamSpec.config) {
         val a1, a2 = TestProbe()
 
         es.subscribe(a1.ref, classOf[Logging.Debug])
+        waitForDebugSubscription(es, a1)
```

Comment:
I think this was the original test case but the same race condition is all over this test, it expects that a subscription is in effect immediately, but it there is a message send inside of the event bus, so `subscribe` returning does not mean the subscription has happened.

The typed event stream test could suffer from the same but has a very (in a strange way) different approach where subscribe and publish and assert messages happens in different testcases across shared probes, so maybe that will always be slow enough to work.